tests/drum/test_custom_model.py

```diff
@@ -1000,10 +1042,6 @@ def test_fit_simple(self, tmp_path):
 
 
 class TestDrumRuntime:
```

Comment:
now when constants and common fixtures are moved to separate files - let's have tests from `TestDrumRuntime` class moved to a separate file (`test_drum_runtime.py`?). No more need to have test class.